Changelog.md

```diff
@@ -1,34 +1,34 @@
-## 0.2.1 - 2024-01-20
+## 0.2.1 - 2024-02-20
 
 ### Added
 
 -   Support for nRF9151 DK.
 -   Better usage data.
--   `Program` step now ensures that device is connected before programming.
 
 ### Changed
 
--   Improved error/notice UI.
+-   Improved error and notice UI.
+-   The Program step, which now ensures that the device is connected before programming.
 
 ### Fixed
 
--   Reading the ICCID value sometimes never finished.
--   Incorrect button label when failing to program.
+-   An issue with reading the ICCID value that would never finish.
```

Comment:
This sounds confusing.
"An issue ... that would never finish"

Reply:
No, "reading the ICCID value that would never finish".
This is the style that was adopted in release notes since nRF5 SDK.